app/views/content/blog/how-to-find-school-experience.md

```diff
@@ -20,6 +20,7 @@ keywords:
 tags:
   - school experience
   - life as a teacher
+closing_paragraph: enriching-the-lives-of-young-people
```

Comment:
Would suggest that rather than adding the generic para, we include the two paras that are already there in the footer signposting ie

* "To start your request for school experience...."
* "If you’re a school or training provider...."